### PR TITLE
implement palindrome checker in python

### PR DESCRIPTION
**Pull request by iattempt**
**Fixes issue:** 
#3113 

**Changes:**

### CODE REVIEW

File: code/utility/palindrome/palindrome_check/palindrome_check.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+def is_palindrome(arr):
```
Comment (arnavb) [inline code/utility/palindrome/palindrome_check/palindrome_check.py]: I remember already implementing this in the past: https://github.com/OpenGenus/cosmos/blob/master/code/string_algorithms/palindrome_checker/palindrome.py. The only thing that needs to be fixed is the recursive implementation.

Comment (arnavb) [reply code/utility/palindrome/palindrome_check/palindrome_check.py]: You could consider moving all the files there to palindrome_check.

Comment (iattempt) [reply code/utility/palindrome/palindrome_check/palindrome_check.py]: I remove my file, and move all palindrome from string_algos. to utiility

**Comment by iattempt** [comment]
The base commit  from this branch is mixed other, I will create new P.R. for this

**Pull request closed without merging**